### PR TITLE
Katriendg/ga v1.0

### PR DESCRIPTION
**Pull request by katriendg**
## Purpose

* Update the sample to use the 1.0.0 release version of AIO and the AIO CLI

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  See readme.

## Other Information

The sample dashboard provided by PG is still missing a few updates which we will import once this is updated from the source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 In this sample, a Visual Studio Code dev container is used as the developer sandbox to leverage a local K3D cluster, initialize Azure resources and install a baseline set of Azure IoT Operations' components.
 
 The objective is to illustrate how to setup observability to be able to locally visualize health of the cluster and Azure IoT Operations' components. This repository is particularly useful for those interested in edge-based visualization. More details about the setup can be found [here](./docs/observability-setup.md).
 
 If you’re looking for a default observability setup and wish to analyze observability data using Azure Arc extensions and Azure Monitor in the cloud, follow [this documentation](https://learn.microsoft.com/en-us/azure/iot-operations/monitor/howto-configure-observability).
 
-> [!WARNING]
-> Azure IoT Operations is still in Preview. Any parts of sample could stop working as the product evolves towards General Availability. The current version is pinned in this dev container is Azure IoT Operations Preview v0.8.32.
+> [!IMPORTANT]
+> This sample is compatible with Azure IoT Operations version 1.0.0.
```
Comment (bindsi) [inline README.md:10]: should we mention the correct version 1.0.9?

File: deploy/3-azure-iot-operations.sh [hunk 2]
```diff
@@ -88,4 +88,4 @@ az deployment group create \
     --parameters customLocationName=$custom_location_name \
     --no-prompt
 
-echo "Finished deploying Azure IoT Operations Preview components to cluster $CLUSTER_NAME in resource group $RESOURCE_GROUP"
\ No newline at end of file
+echo "Finished deploying Azure IoT Operations ${cli_pinned_version} components to cluster $CLUSTER_NAME in resource group $RESOURCE_GROUP"
```
Comment (bindsi) [inline deploy/3-azure-iot-operations.sh:91]: is this really the version or only the CLI version? just to be aligned with my version comment above

**Review by bindsi — APPROVED** [review]
Thanks for updating Katrien! lgtm

File: deploy/3-azure-iot-operations.sh (hunk 2 above)
Comment (katriendg) [reply deploy/3-azure-iot-operations.sh:91]: Confusing, so for CLI it's version 1.0.0 (there is no 1.0.9) - https://github.com/Azure/azure-iot-ops-cli-extension/releases/tag/v1.0.0. 
For the 'AIO GA' release we see 1.0.9 - https://github.com/Azure/azure-iot-operations/releases but this does contain anything other than the ARM templates used by the CLI. I would suggest we keep it on par with the CLI. Wdty?

File: README.md (hunk 1 above)
Comment (katriendg) [reply README.md:10]: Maybe here I could use 1.0?

File: deploy/3-azure-iot-operations.sh (hunk 2 above)
Comment (bindsi) [reply deploy/3-azure-iot-operations.sh:91]: oh yes, let´s keep 1.0.0 at all, less confusing when somebody checks the CLI version.

Thanks Katrien!

Comment (katriendg) [reply deploy/3-azure-iot-operations.sh:91]: Thank you Marcel :)

**Pull request merged**